extensions/2.0/Khronos/KHR_gaussian_splatting_compression_spz_2/README.md

```diff
@@ -0,0 +1,165 @@
+# KHR\_gaussian\_splatting\_compression\_spz\_2
+
+## Contributors
+
+- Jason Sobotka, Cesium
+- Renaud Keriven, Cesium
+- Adam Morris, Cesium
+- Sean Lilley, Cesium
+- Projit Bandyopadhyay, Niantic Spatial
+- Daniel Knoblauch, Niantic Spatial
+- Ronald Poirrier, Esri
+- Jean-Philippe Pons, Esri
+
+## Status
+
+Draft
+
+## Dependencies
+
+Written against the glTF 2.0 spec.
+
+Depends on the `KHR_gaussian_splatting` extension specification for attribute definitions.
+
+## Table of Contents
+
+- [Overview](#overview)
+- [Compressing 3D Gaussian splats using SPZ](#compressing-3d-gaussian-splats-using-spz)
+  - [Geometry Type](#geometry-type)
+  - [Schema Example](#schema-example)
+  - [Extension Properties](#extension-properties)
+  - [Accessors](#accessors)
+- [Conformance](#conformance)
+- [Schema](#schema)
+- [Known Implementations](#known-implementations)
+- [Resources](#resources)
+
+## Overview
+
+This extension defines support for compressing 3D Gaussian splats stored within in glTF using [v2 of the SPZ compression format](https://github.com/nianticlabs/spz/tree/v2.0.0) for efficient streaming and storage.
+
+SPZ is a compression format from Niantic Spatial designed for Gaussian splats. Currently, it is open sourced under the MIT license. The SPZ format is primarily used in the Niantic Spatial Scaniverse app. It was purpose built for Gaussian splats and offers a balance of high compression with minimal visual fidelity loss, and allows for storing some or all of the 3D Gaussian's spherical harmonics.
+
+At rest, the 3D Gaussian splats are stored within the SPZ compression format. Upon being decompressed, values are mapped to the glTF attributes defined by `KHR_gaussian_splatting` for rendering. The [conformance](#conformance) section defines what an implementation must do when encountering this extension, and how the extension interacts with the base specification.
+
+## Compressing 3D Gaussian splats using SPZ
+
+To use this extension, it must be defined in the `extensions` property in a `KHR_gaussian_splatting` extension definition. Any mesh primitive using `KHR_gaussian_splatting` that is using this extension will use the SPZ payload to retreive the values for `POSITION`, `COLOR_0`, `KHR_gaussian_splatting:SCALE`, `KHR_gaussian_splatting:ROTATION`, and all `KHR_gaussian_splatting:SH_DEGREE_ℓ_COEF_n` attributes. The [attribute mapping](#attribute-mapping) section defines how the SPZ data is mapped to these attributes.
+
+The extension must then be listed in `extensionsUsed` alongside `KHR_gaussian_splatting`.
+
+```json
+  "extensionsUsed" : [
+    "KHR_gaussian_splatting",
+    "KHR_gaussian_splatting_compression_spz"
+  ]
+```

```

Comment:
If fallback data is not provided then the extension should also go in `extensionsRequired`, similar to the [Draco extension](https://github.com/KhronosGroup/glTF/tree/main/extensions/2.0/Khronos/KHR_draco_mesh_compression#gltf-schema-updates):

<img width="1430" height="619" alt="image" src="https://github.com/user-attachments/assets/aefd3648-8e72-4c75-a9a1-32699eae75cf" />